manifests/base/crds/argoproj.io_eventbuses.yaml

```diff
@@ -0,0 +1,40 @@
+# This is an auto-generated file. DO NOT EDIT
```

Comment:
Why is this generated?

Reply:
No idea...
I ran `make codegen` but since this is a new file,  I guess the `git diff` in the CI did not pick this up.
Removed it, OK?